### PR TITLE
Add documentations and remove API key from codes

### PR DESCRIPTION
**Pull request by ellennugroho**
Finished documentations on readme and use .env for storing API key for `examples`.

### CODE REVIEW

**Comment by ellennugroho** [comment]
I'm still waiting for `balance` to be merged before this PR is ready.

File: .env.example [hunk 1]
```diff
@@ -0,0 +1 @@
+SECRET_API_KEY = "insert your secret API key here"
```
Comment (kevindavee) [inline .env.example]: Maybe don't add `"` in the example? I'm afraid it will be misleading

File: examples/CardsExample.php [hunk 2]
```diff
@@ -15,11 +15,10 @@
...
 
-Xendit::setApiKey(
-    <<<'TAG'
-xnd_development_prHUBDfVuOQTxyWTQSNkpjn9OwX9ZSUjdqgF9GenZ6hwhUQkc3NZ9WVexdH
-TAG
-);
+$dotenv = Dotenv\Dotenv::createImmutable(__DIR__ . '/..');
+$dotenv->load();
+
+Xendit::setApiKey(getenv('SECRET_API_KEY'));
```
Comment (kevindavee) [inline examples/CardsExample.php]: Is it possible to extract the logic into a different function? So it will not create duplication.

Comment (ellennugroho) [reply examples/CardsExample.php]: yes, it's possible. thanks for the advice! code fix: [here](https://github.com/xendit/xendit-php-clients/pull/47/commits/d3b502d9051ff16effc2a98ce42eda72c57b4ac9)

File: .env.example (hunk 1 above)
Comment (ellennugroho) [reply .env.example]: so it should be only `SECRET_API_KEY=` ?

Comment (ellennugroho) [reply .env.example]: or `SECRET_API_KEY=""`?

File: .env.example [hunk 3]
```diff
@@ -0,0 +1 @@
+SECRET_API_KEY =
```
Comment (kevindavee) [inline .env.example]: This one is fine. Or if you want to do it this way, it is also fine
`SECRET_API_KEY=your_api_key`

**Pull request merged**